draft-ietf-ohai-ohttp.md

```diff
@@ -132,75 +132,84 @@ them or linked to other requests.
 
 A client must initially know the following:
 
-* The identity of an oblivious request resource.  This might include some
-  information about what oblivious target resources the oblivious request
+* The identity of an Oblivious Gateway Resource.  This might include some
+  information about what Oblivious Target Resources the Oblivious Gateway
   resource supports.
```

Comment:
Let's talk capiTaLiZaTiOn.

I was fairly uniform in terms of not being overly Germanic.  These are not proper nouns, just names.  But you seem to have taken the opposite approach.

I prefer all lowercase, but I don't object to lending the names a little pomp and grandeur either.

You missed a s/resource/Resource here, that's all.

Reply:
Good catch -- fixed.